package/docker/Dockerfile.ubuntu-bionic

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:bionic
+
+ARG USER_ID=1000
```

Comment:
the same problem with ARG exists here.

Reply:
Same response. I like arguments at the top of the file, and don't think this will reduce cacheing.

Reply:
Moved it back.